src/deluge/storage/cluster/cluster.cpp

```diff
@@ -232,3 +232,12 @@ bool Cluster::mayBeStolen(void* thingNotToStealFrom) {
 	}
 	return true;
 }
+
+void Cluster::addReason() {
+	// If it's going to cease to be zero, it's become unavailable
```

Comment:
since this is no longer used to hackily remove things from the loading queue should we clarify the comment to specify that it's being removed from its stealable queue?

Reply:
done!